connector/src/main/java/org/jboss/as/connector/subsystems/jca/JcaExtension.java

```diff
@@ -63,7 +63,7 @@ public class JcaExtension implements Extension {
 
     public static final String SUBSYSTEM_NAME = "jca";
 
-    private static final ModelVersion CURRENT_MODEL_VERSION = ModelVersion.create(6, 0, 0);
+    private static final ModelVersion CURRENT_MODEL_VERSION = ModelVersion.create(6, 1, 0);
```

Comment:
This jira appears to be specific to XML parsing.  For what reason are you incrementing the model version?

Reply:
@tadamski are the schema and model version not the same thing? I'm increasing the schema version, same as #17866

Reply:
They are not the same thing.
One can change the subsystem model without affecting the XML schema (like you are doing here), just as one can change the XML schema without affecting the subsystem model.